src/Controls/src/BindingSourceGen/SetterBuilder.cs

```diff
@@ -68,6 +67,11 @@ public void AddPart(IPathPart nextPart)
                     AddIsExpression("{}");
                     _expression = AccessExpressionBuilder.Build(_expression, innerPart);
                 }
+                else if (previousPart is MemberAccess memberAccess && !memberAccess.IsMemberValueType && _considerAllReferenceTypesPotentiallyNullable) 
+                {
+                    AddIsExpression("{}");
+                    _expression = AccessExpressionBuilder.Build(_expression, memberAccess);
+                }
```

Comment:
I don't think this is equivalent to the removed condition in `Setter.From`. If this didn't break any existing test though, I need to think a bit if we are missing a test scenario or not.

Reply:
This is still WIP. The previous version contained a bug. When we had nullable source and first access in the path was conditionalAccess it generated `if p0 is {} p1`.

Reply:
oh, good catch!